src/components/date-picker/DatePicker.vue

```diff
@@ -17,10 +17,10 @@
       bottom
       min-width="290"
     >
-      <template #activator="{ on }">
```

Comment:
The `v-menu` would not open with `v-on` and `on`. Using `props` seems to work

Reply:
Did you see this? Does it help? https://stackoverflow.com/questions/75959657/replacement-for-vue-2-v-on-in-vue-3